src/gasman.c

```diff
@@ -1140,6 +1140,13 @@ void SetExtraMarkFuncBags(TNumExtraMarkFuncBags func)
 
 GAP_STATIC_ASSERT((sizeof(BagHeader) % sizeof(Bag)) == 0, "BagHeader size must be multiple of word size");
 
+
+void MarkStackBottomBags(void * StackBottom)
```

Comment:
I just realized that `MarkStackBottomBags` is a bad name, because it evokes a relation to the other `MarkFOO` functions. A better name (in the context of GAP and its naming conventions) would be `InitStackBottomBags`, or perhaps `SetStackBottomBags`, `SetupStackBottomBags`

Reply:
Okay, renamed `MarkStackBottomBags` ➡️ `SetStackBottomBags`.